api/models/usersChannels.js

```diff
@@ -669,6 +671,29 @@ const getUserUsersChannels = (userId: string) => {
     .run();
 };
 
+const getUserChannelIds = (userId: string) => {
+  return db
+    .table('usersChannels')
+    .getAll(userId, { index: 'userId' })
+    .filter({ isMember: true })
+    .map(rec => rec('channelId'))
+    .run();
```

Comment:
Does this then need a `.then(cursor => cursor.toArray())` ? I can't remember exactly when that's needed

Reply:
rethinkdbdash automatically converts cursors to arrays, only necessary in migrations!